tutorials/32_Language_Classification_In_Documents_And_Queries.ipynb

```diff
@@ -0,0 +1,1501 @@
+{
```

Comment:
<p>Link to DocumentLanguageClassifier doc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>